neurom/core/_soma.py

```diff
@@ -44,8 +44,10 @@ class Soma(object):
     Holds a list of raw data rows corresponding to soma points
     and provides iterator access to them.
     '''
+
     def __init__(self, points):
         self._points = points
+        self.radius = 0
```

Comment:
I added the radius attribute here as I discovered that otherwise you can't call the draw method on a Neuron based on a Soma instance (instead of let's say SomaSinglePoint for example).

Reply:
yup, makes sense